pkg/client/portforward/portforward_test.go

```diff
@@ -206,6 +206,102 @@ func (s *fakeUpgradeStream) Headers() http.Header {
 	return http.Header{}
 }
 
+func TestGetListener(t *testing.T) {
+	stopChan := make(chan struct{}, 1)
+	pf, err := New(&client.Request{}, &client.Config{}, []string{"15000", "15000"}, stopChan)
+	if err != nil {
+		t.Logf("Cannot create PortForwarder %v: %v", pf, err)
+
+	}
+	upgrader := fakeUpgrader{conn: newFakeUpgradeConnection()}
+	pf.upgrader = &upgrader
+	testCases := []struct {
+		Hostname                  string
+		Protocol                  string
+		Ports                     ForwardedPort
```

Comment:
I don't see any need to vary the ports in the test cases. I'd recommend hardcoding something inside the for loop instead.